tools/scripts/docs_modm_io_index.html.in

```diff
@@ -85,24 +84,26 @@ input[type=submit] {
 <h1>API documentation</h1>
 </header>
 <main>
-<p id="numtargets">Documentation is available for {{ num_devices }} devices.</p>
+<p id="numtargets">Documentation is available for {{ num_devices }} devices and {{ num_boards }} boards.</p>
 <h3>Select your target:</h3>
 <form autocomplete="off" action="javascript:showDocumentation()">
 <table>
     <tr>
-        <td>Version/Release</td>
-        <td>Microcontroller</td>
+        <!-- <td>Version/Release</td> -->
+        <td>Device or Board</td>
         <td>&nbsp;</td>
     </tr>
     <tr>
+        <!--
         <td>
             <select id="releaseinput" name="releases">
                 <option>develop</option>
             </select>
         </td>
+        -->
         <td>
             <div class="autocomplete" style="width:300px;">
-                <input id="targetinput" type="text" name="target" placeholder="Type e.g. 'F407'">
+                <input id="targetinput" type="text" name="target" placeholder="Type e.g. 'F407' or 'Blue Pill'">
```

Comment:
```suggestion
                <input id="targetinput" type="text" name="target" placeholder="Type e.g. 'F407' or 'Nucleo-G474'">
```

Not everyone immediately recognizes "Blue Pill" as a dev board....

Reply:
Perhaps just `Search for device or board` and the people just know it's a search field?

Reply:
Nah, having an example is always better.

Reply:
Yeah, it also just duplicates the description, which already says `Device or board`.